app/controllers/Janus.scala

```diff
@@ -239,7 +238,7 @@ class Janus(
         janusData.access
       )
     } yield {
-      AuditTrailDB.insert(table, auditLog)
+      AuditTrailDB.insert(auditLog)
```

Comment:
I realise this isn't part of your change, but while we're here anyway and tidying up the DB usage can we pull this up into the for comp steps? I don't like that we have an important side-effecting step in the yield block here (my apologies for the current state of things).

Something like `_ = AuditTrailDB.insert(auditLog)` as the last step would highlight that this is an important operation, separate from building our return value.

I'd do the same for the log line, but I don't feel as strongly we should do that now.

Equally, I'm very happy to tackle this along with a wider review of the codebase in a separate PR - I only mention it here because our diff has hit it.

Reply:
Done in d8d0f9e8c1a6ac653e3dd61d31cdabf48d4dc32c